scripts/dev.ts

```diff
@@ -49,14 +55,31 @@ class DevRunner extends events.EventEmitter {
     return promise.child;
   }
 
+  /**
+   * Gets information about the renderer based on the environment variable
+   * RD_ENV_PLUGINS_DEV. For plugins development.
+   */
+  rendererEnv(): RendererEnv {
+    if (process.env.RD_ENV_PLUGINS_DEV) {
+      return {
+        home:  'https://localhost:8888/home',
+        agent: new https.Agent({ rejectUnauthorized: false }),
+      };
+    }
+
+    return { home: 'http://localhost:8888/pages/General' };
+  }
+
   #mainProcess: childProcess.ChildProcess | null = null;
   async startMainProcess() {
     try {
       await buildUtils.buildMain();
+      const rendererEnv = this.rendererEnv();
+
       // Wait for the renderer to finish, so that the output from nuxt doesn't
       // clobber debugging output.
       while (true) {
-        if ((await fetch('http://localhost:8888/pages/General')).ok) {
+        if ((await fetch(rendererEnv.home, { agent: rendererEnv.agent })).ok) {
```

Comment:
Looking at the docs for `node-fetch`, the default value for `agent` is `null`, not `undefined`. Obviously it's working as is, but I'd change the code to `{ agent: rendererEnv.agent ?? null }`

Reply:
Attempting to make the suggested change produces the following error

```
Type 'Agent | null' is not assignable to type 'Agent | ((parsedUrl: URL) => Agent) | undefined'.
```

After further inspecting `@types/node-fetch` shows `agent?: Agent | ((parsedUrl: URL) => Agent) | undefined`. Perhaps there's a difference between the version documented and what we run?

Reply:
Hmm I didn't test the change, was just reading the docs. Fine to leave it as is.